helm/botkube/values.yaml

```diff
@@ -417,10 +417,47 @@ resources: {}
   #  cpu: 100m
```

Comment:
Continuing what @PrasadG193 has asked about, I really like the PR testing instruction and I think it would be great as a separate tutorial on `botkube.io`. Apart from updating docs when it comes to Helm chart properties, please also copy the instruction there 🙂 

BTW and I think we could have a separate branch (e.g. `develop`) for now, until there are no docs versioning on botkube.io site. Feel free to create one and open the PR on `botkube-docs`, I will review it 🙂

Reply:
Yes, that's what I also wanted to discuss. 

IMO we can convert the [BotKube external secret cfg](https://github.com/infracloudio/botkube/issues/480#issuecomment-1139559186) comment into a generic document. Additionally, add a dedicated section with a `BotKube with Vault via CSI driver` use-case. 

WDYT @PrasadG193 @pkosiec ?

Reply:
yep, a dedicated document sounds the best for me 👍